### DISCUSSION TITLE
Correct way of adding extra arguments?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [hlky/stable-diffusion-webui#802] [Q&A]
![Screenshot_7](https://user-images.githubusercontent.com/13344308/189008537-d76cb380-161b-41ec-811e-0f1e366a0c7d.jpg)

![Screenshot_6](https://user-images.githubusercontent.com/13344308/189008425-e44557c1-29a8-4f70-bde0-543d99444e8e.jpg)

It's the correct sintaxis?

**Comment by edgartaor** [comment]
I think is: 
`additional_arguments = "--precision=full --another_argument=9"`
